deploy/stacks/container.py

```diff
@@ -260,7 +260,7 @@ def add_share_management_task(self):
             f'ShareManagementTaskContainer{self._envname}',
             container_name='container',
             image=ecs.ContainerImage.from_ecr_repository(repository=self._ecr_repository, tag=self._cdkproxy_image_tag),
-            environment=self._create_env('DEBUG'),
+            environment=self.env_vars,
```

Comment:
Adding self.env_vars as they contain necessary configs for email notification code to work. 
Email notifications are sent if attachable managed IAM policy quota is exceeded

Reply:
Aren't `self.env_vars = self._create_env()` ? Does this change change anything? It just replaces the calling the function by using the variable directly no?

Reply:
self.env_vars are modified during the initialization code to include the email notification env variables. Function _create_env doesn't have those so I used self.env_vars